plugins/net_plugin/net_plugin.cpp

```diff
@@ -3932,7 +3934,7 @@ namespace eosio {
            "Number of worker threads in net_plugin thread pool" )
          ( "sync-fetch-span", bpo::value<uint32_t>()->default_value(def_sync_fetch_span), "number of blocks to retrieve in a chunk from any individual peer during synchronization")
          ( "use-socket-read-watermark", bpo::value<bool>()->default_value(false), "Enable experimental socket read watermark optimization")
-         ( "peer-log-format", bpo::value<string>()->default_value( "[\"${_name}\" ${_ip}:${_port}]" ),
+         ( "peer-log-format", bpo::value<string>()->default_value( "[\"${_name}\" ${_ip}:${_port}] " ),
```

Comment:
undo this unneeded diff

Reply:
I did this because of default format lacks space and it harder to read.
so now it looks like this:
`["localhost:9877" 127.0.0.1:9877] preverified: [true] certificate subject: /C=US/ST=VA/L=Blockburg/O=node2/CN=test-domain2`
not like this:
`["localhost:9877" 127.0.0.1:9877]preverified: [true] certificate subject: /C=US/ST=VA/L=Blockburg/O=node2/CN=test-domain2`